### PR TITLE
Telescopic shield

### PR DESCRIPTION
**Pull request by Nimfar11**
## About the PR

Telescopiс shield for HoS, more convenient and compact than a regular riot shield, as well as slightly durable.

Moved the sprites from another folder so you can overlay the visuals.

## Media

Telescopic shield

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

:cl: Nimfar11

- add: Added a telescopiс shield to the Advanced Riot Control Technology.

### CODE REVIEW

**Comment by Nimfar11** [comment]
Is sound a must, or can we do without it?

**Comment by HoidC** [comment]
isn't the HoS armour already good, do they need another damage resistance buff on top of that

**Comment by Nimfar11** [comment]
> isn't the HoS armour already good, do they need another damage resistance buff on top of that

It's a slightly more durable shield than the others have, its most important advantage, HoS can carry it in his pocket until he uses it.

**Comment by LankLTE** [comment]
The HoS already gets a gun, armor, and suit. They don’t need more unique and good stuff just because they’re a head. The item could work well as a sci research, though, and I’d encourage adding it there.

**Comment by Nimfar11** [comment]
Okay, I see the shield for HoS didn't come up. If there are no new suggestions, I'll make it as a technology.

**Comment by Nairodian** [comment]
Yeah I'd love to see a more versatile shield that isn't just restricted to the HoS. I agree in making it a technology.

**Comment by deltanedas** [comment]
this would go very well in advanced riot control with the truncheon

**Comment by HerCoyote23** [comment]
That was actually the intent when I made Advanced Riot Control and truncheon: I was planning on making teleshield and putting it in there. I'd recommend that you add it to Advanced Riot Control.

**Comment by Nimfar11** [comment]
Yeah, that's what I thought. The truncheon in technology seems lonely.

**Comment by JoeHammad1844** [comment]
dont raise the price.
i support adding the idea of adding it as a tech but keep one in his locker

**Comment by NancokPS2** [comment]
Wouldn't it be more logical to make it less durable?  
That way you have a shield with unique advantages that is not a direct upgrade.

**Comment by LankLTE** [comment]
> Wouldn't it be more logical to make it less durable?  
> 
> That way you have a shield with unique advantages that is not a direct upgrade.  

If it's a Sci research it can be a direct upgrade

File: Resources/Prototypes/Entities/Objects/Shields/shields.yml [hunk 1]
```diff
@@ -400,3 +400,65 @@
...
+- type: entity
+  name: telescopic shield
+  parent: BaseShield
+  id: TelescopicShield
+  description: An advanced riot shield made of lightweight materials that collapses for easy storage.
+  components:
+    - type: ItemToggle
+      activatedDisarmMalus: 0.6
+      activateSound: false
+      deActivateSound: false
```
Comment (LankLTE) [inline Resources/Prototypes/Entities/Objects/Shields/shields.yml:413]: Add a sound or remove this.

**Review by LankLTE — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Shields/shields.yml (hunk 1 above)
Comment (Nimfar11) [reply Resources/Prototypes/Entities/Objects/Shields/shields.yml:413]: If you do not install these components, the Client crashes when trying to look at the shield. I'll try to find the sound then.

File: Resources/Audio/Weapons/attributions.yml [hunk 2]
```diff
@@ -28,3 +27,8 @@
   license: "CC-BY-SA-3.0"
   copyright: "Based on genhit3.ogg from tgstation, modified by deltanedas on github."
   source: "https://github.com/deltanedas"
+
+- files: ["telescopicon.ogg, telescopicoff.ogg"]
+  license: "CC-BY-SA-3.0"
+  copyright: "User Nimfar11 on GitHub."
+  source: "https://github.com/Nimfar11"
```
Comment (LankLTE) [inline Resources/Audio/Weapons/attributions.yml:34]: Did you make this sound?

Comment (Nimfar11) [reply Resources/Audio/Weapons/attributions.yml:34]: Handbrake in the car... not my car.

File: Resources/Prototypes/Entities/Objects/Shields/shields.yml [hunk 3]
```diff
@@ -400,3 +400,71 @@
...
+            - !type:PlaySoundBehavior
+              sound: /Audio/Effects/metalbreak.ogg
+            - !type:SpawnEntitiesBehavior
+              spawn:
+                SheetSteel:
+                  min: 1
+                  max: 2
+                SheetGlass:
+                  min: 2
+                  max: 3
```
[the remark this answers is not in the document]
Comment (Nimfar11) [reply Resources/Prototypes/Entities/Objects/Shields/shields.yml]: done

[the remark this answers is not in the document]
Comment (LankLTE) [reply Resources/Prototypes/Entities/Objects/Shields/shields.yml]: Man what are these tests on

**Review by LankLTE — APPROVED** [review]

**Pull request merged**